content/t/slides/contents.lr

```diff
@@ -0,0 +1,3 @@
+_model: redirect
+---
+new_path: https://www.dropbox.com/scl/fi/h6zngysnbibk3ly4e0qz9/Build-a-cross-platform-GUI-app-with-BeeWare.pdf?rlkey=knjrqcv5e6wxzv9335va0j5qu&st=t1t4ppji&dl=0
```

Comment:
This redirect doesn't work because it replaces each `&` in the URL with `&amp;`.